style-guide/fields.md

```diff
@@ -0,0 +1,7 @@
+# Field Value Subsection
```

Comment:
I wanted to put the Field Value and the Property Value all in one file along with the Return Value information.

Reply:
Ah, sorry about that - I thought it was a typo. Fixing.